packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -0,0 +1,170 @@
+import { commonPropTypes, createShorthandFactory, UIComponentProps } from '../../utils';
+import Grid from '../Grid/Grid';
+import {
+  IDayGridOptions,
+  getDayGrid,
+  IDay,
+  DAYS_IN_WEEK,
+  IDateGridStrings,
+  IDateFormatting,
+  DayOfWeek,
+  FirstWeekOfYear,
+  DateRangeType,
+} from '@fluentui/date-time-utilities';
+import Text from '../Text/Text';
+import Button from '../Button/Button';
+import { IDatepickerOptions } from './Datepicker';
+
+export interface DatepickerCalendarProps extends IDatepickerOptions, IDateFormatting, UIComponentProps {
+  /**
+   * The currently selected date
+   */
+  selectedDate: Date;
+  /**
+   * The currently navigated date
+   */
+  navigatedDate: Date;
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<DatepickerCalendarBehaviorProps>;
+  /** Callback on date cell selection */
+  onDaySelect?: (IDay) => void;
+  /** Localized labels */
+  localizedStrings?: IDateGridStrings;
+}
+
+export type DatepickerCalendarStylesProps = never;
+
+export const datepickerCalendarClassName = 'ui-datepickerCalendar';
+
+const DatepickerCalendar: React.FC<WithAsProp<DatepickerCalendarProps>> &
+  FluentComponentStaticProps<DatepickerCalendarProps> = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(DatepickerCalendar.displayName, context.telemetry);
+  setStart();
+  const datepickerCalendarRef = React.useRef<HTMLElement>();
+  const {
+    className,
+    design,
+    styles,
+    variables,
+    selectedDate,
+    navigatedDate,
+    firstDayOfWeek,
+    firstWeekOfYear,
+    dateRangeType,
+    onDaySelect,
+    localizedStrings,
+  } = props;
+  const ElementType = getElementType(props);
+  const unhandledProps = useUnhandledProps(DatepickerCalendar.handledProps, props);
+  const getA11yProps = useAccessibility(props.accessibility, {
+    debugName: DatepickerCalendar.displayName,
+    actionHandlers: {},
+    rtl: context.rtl,
+  });
+  const gridOptions: IDayGridOptions = {
+    selectedDate,
+    navigatedDate,
+    firstDayOfWeek,
+    firstWeekOfYear,
+    dateRangeType,
+  };
+
+  const { classes } = useStyles<DatepickerCalendarStylesProps>(DatepickerCalendar.displayName, {
+    className: datepickerCalendarClassName,
+    mapPropsToInlineStyles: () => ({
+      className,
+      design,
+      styles,
+      variables,
+    }),
+    rtl: context.rtl,
+  });
+
+  let grid = getDayGrid(gridOptions);
+  // Slicing because grid contains extra 1 week in the front and in the back.
+  grid = grid.slice(1, grid.length - 1);
+
+  const element = (
+    <Ref innerRef={datepickerCalendarRef}>
+      {getA11yProps.unstable_wrapWithFocusZone(
+        <ElementType
+          {...getA11yProps('root', {
+            className: classes.root,
+            ...unhandledProps,
+          })}
+        >
+          <Grid rows={grid.length + 1} columns={grid[0].length}>
```

Comment:
```suggestion
          <Grid rows={grid.length + 1} columns={DAYS_IN_WEEK}>
```
I'd suggest either to have here the same number as we use in `_.times` iterator on the next line. 
Can be DAYS_IN_WEEK constant or grid[0].length, I'm OK with either :)

Reply:
Fixed.